src/main/java/codeu/model/store/basic/DefaultDataStore.java

```diff
@@ -102,7 +102,7 @@ private void addRandomUsers() {
 
     for (int i = 0; i < DEFAULT_USER_COUNT; i++) {
         User user = new User(UUID.randomUUID(), randomUsernames.get(i), Instant.now(), 
-          BCrypt.hashpw("password", BCrypt.gensalt()));
+          BCrypt.hashpw("password", BCrypt.gensalt()), getRandomAboutContent());
```

Comment:
What does `hashpw` do? Why are we hashing the default about content with the password?

Reply:
hashPW creates a hashed password. We aren't hashing the default about content, but the getRandomAboutContent() is an argument passed in to create a new User.

Reply:
But why is it necessary to use the random about content to create the hashed password? Did we use the user's password to create the hashed password previously, and if so, why don't we continue doing that?

Reply:
The getRandomAboutContent() call isn't used to create the hashed password. It's just another argument passed to create a user, like UUID.randomUUID() or Instant.now(). BCrypt.hashpw("password", BCrypt.gensalt()) returns a string that is a hashed password

Reply:
Cool, I think `hashpw` and `getRandomAboutContent` make sense to me individually! I'm just confused why we need to pass in the return value of `getRandomAboutContent()` into `hashpw()` when it was generating hashed passwords fine without the about content before, too.

Reply:
Ooooh I don't think I'm passing `getRandomAboutContent()` into `hashpw()`?

Reply:
I think your pull request shows that you've modified line 105 to include `getRandomAboutContent()` as the last parameter to `hashpw()`!

Reply:
Generally I would just change getRandomAboutContent() to a constant string, which you can then later check if it was set on the User correctly in datastore.